transport/person.go

```diff
@@ -0,0 +1,93 @@
+package transport
+
+import (
+	"bufio"
+	"encoding/csv"
+	"errors"
+	"github.com/labstack/echo/v4"
+	"github.com/pankecho/golang-bootcamp-2020/entity"
+	"io"
+	"net/http"
+	"os"
+	"time"
+)
+
+type PersonUseCase interface {
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Please add a description for every exported thing here